### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by vargas055**
Revised functions and changed var to let

### CODE REVIEW

**Comment by vargas055** [comment]
@mpstanford @mbbph @dbooyah93  forgot to tag you all :)

File: Server/app.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ const Review = require('../database-mongodb/Review.js');
 
 //MIDDLEWARE
 app.use(bodyParser.json())
-// if unsure how the code is working or what it requires check this gist https://gist.github.com/vargas055/3173a744411f52a68f5d8e8c61f3da3c
+// if unsure how the code is working or what it requires check this gist https://gist.github.com/letgas055/3173a744411f52a68f5d8e8c61f3da3c
```
Comment (dbooyah93) [inline Server/app.js:9]: I tried going to that link with a 404 error appearing. Maybe it's private?

File: globalConfig.json [hunk 2]
```diff
@@ -1 +1 @@
-{"mongoUri":"mongodb://127.0.0.1:55183/reviewesModule?","mongoDBName":"reviewesModule"}
\ No newline at end of file
+{"mongoUri":"mongodb://127.0.0.1:51279/reviewesModule?","mongoDBName":"reviewesModule"}
```
Comment (dbooyah93) [inline globalConfig.json:1]: On windows to end a connection to a server the short cut is `ctrl + c` but if I hit `ctrl + z` the server stops running in the terminal, but still runs on the computer memory and will persist until the computer is shut down. Idk if you are running into the error "server in use" but if you are consider how you close your servers

**Review by dbooyah93 — COMMENTED** [review]
Looks good over all, I added some little bits of info about your changed addresses. Check your gist is public, and if you are running into "server in use" errors, check my second comment. Nice switch to `let`! :D

File: globalConfig.json (hunk 2 above)
Comment (vargas055) [reply globalConfig.json:1]: that mongo uri is for the database but i am closing those connections after the seed and after the tests so so far no error, but ill keep in mind if do get errors!

**Pull request merged**